LICENSE

```diff
@@ -31,13 +32,13 @@ License: Apache Software License Version 2.0
 Not packaged with release; development using version 1.6.x
 
 iText (PDF generation)
-http://www.lowagie.com/iText/ and http://itextpdf.sourceforge.net/
+http://itextpdf.com/
 License: Mozilla Public License Version 1.1
-Using version 2.0.8.
-Included as lib/iText-2.0.8.jar
+Using version 2.1.7.
+Included as lib/iText-2.1.7.jar
```

Comment:
I think we should also mention itext5 here, which is available under a different license (AGPL).